packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -75,6 +77,7 @@ const firmwareInstall =
         if (fwBinary) {
             console.warn(`Installing custom firmware`);
 
+            // todo: what about firmware hash ?
```

Comment:
@tomasklim

Reply:
surely we can and want to add it, but as a separate task